lib/controllers.js

```diff
@@ -19,8 +19,6 @@ module.exports = function (rootDirectory, logger, indexDocument, errorDocument,
 
   var buildResponse = function (req, res, status, object, data) {
     res.header('Access-Control-Allow-Origin', '*');
-    res.header('Etag', '"' + object.md5 + '"');
-    res.header('Last-Modified', new Date(object.modifiedDate).toUTCString());
```

Comment:
Does your addition work without removing these lines? Remove these lines will cause `Etag` and `Last-Modified` values to disappear in other calls to  `buildResponse`.